src/app/static/src/tests/components/password/forgotPasswordPage.test.ts

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+const path = require('path');
+const html = fs.readFileSync(path.resolve(__dirname, '../../../../templates/forgot-password.ftl'), 'utf8');
+
+document.documentElement.innerHTML = html.toString();
```

Comment:
Very cunning! Just wonder if this should go into a different root test folders than 'components' since it's not actually testing a component. 'pages'  or 'templates'. No doubt we'll make other tests like this, and they should probably be kept together.

Reply:
Good shout